ccloud-observability/docs/general-scenarios/request-rate-limits.rst

```diff
@@ -0,0 +1,32 @@
+.. _ccloud-observability-general-request-rate-limits:
+
+Request rate limits
+********************
+
+If request rate limits are hit, requests may be refused and clients may be throttled to keep the cluster stable. This throttling
+would register as non-zero values for the producer client ``produce-throttle-time-max`` and
+``produce-throttle-time-avg`` metrics and consumer client ``fetch-throttle-time-max`` and ``fetch-throttle-time-avg`` metrics.
+
+#. Open `Grafana <localhost:3000>`__ and use the username ``admin`` and password ``password`` to login.
+
+#. Navigate to the ``Confluent Cloud`` dashboard.
+
+#. Check the ``Requests (rate)`` panel. If this panel is yellow, you have used 80% of your allowed requests; if it's red, you have used 90%.
```

Comment:
IIUC, these thresholds are defined in `"monitoring_configs/grafana/provisioning/dashboards/ccloud-exporter.json` - can you point users to this code so they can learn where to set these if they want to tweak it for themselves?

Reply:
The best place is to do it through the UI. I have left it out thus far because I feel like this walks the line of teaching people how to use grafana rather than how to use Confluent, I plan to go into detail about thresholds in my blog post, and there are multiple places where thresholding is mentioned. 

All this being said, I understand the desire to talk about how we configured these thresholds, so I can one or a combination of the following:
1. Create an "includes" that walks the user through changing thresholds, add literalinclude to easily reference with each mention of thresholding. 
2. Add link to grafana docs on tresholding like "   See `Grafana documentation <https://grafana.com/docs/grafana/latest/panels/thresholds/>`_ for more information about about configuring thresholds.  
"
3. Point to additional resources page where grafana docs and future blog post will be. 
4. <did you have something else in mind?>

Reply:
@awalther28 great point.  I think we don't want to go down the slippery slope of Grafana teaching.

I think the question I had in my mind was "Where is this 80%/90% limit set?"  Specifically, users might think it's something set by CCloud Metrics API somehow...which it's not.  It's customizable and set by the dashboard config.  So maybe we can address this a simple way -- the option #2 seems really good to me.